ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -72,7 +72,21 @@ - (void)updateMap
         _map.scrollEnabled = _scrollEnabled;
         _map.zoomEnabled = _zoomEnabled;
         _map.showsUserLocation = _showsUserLocation;
-        _map.styleURL = _styleURL;
+        if ([_styleURL  isEqual: @"lightStyleURL"]) {
```

Comment:
Instead of accepting magic strings here, there should be a separate JavaScript object that vends string constants that can be set as the styleURL.